lexicons/com/atproto/admin/getInviteCodeUsage.json

```diff
@@ -0,0 +1,32 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.admin.getInviteCodeUsage",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "High level stats about invite code usage",
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["total", "user", "admin"],
+          "properties": {
+            "total": { "type": "ref", "ref": "#codesDetail" },
+            "user": { "type": "ref", "ref": "#codesDetail" },
+            "admin": { "type": "ref", "ref": "#codesDetail" }
```

Comment:
I think it's on the horizon to switch-up how admin auth works, and certain user accounts may be marked as admins.  Just tossing that out there for a gut check that this API will still match-up with that nicely, given that there could be overlap between "user" and "admin".

I think it still could work— "user" would just have to be specifically for non-admin accounts.  Or another way to think about it is whether the invite code is generated from `createInviteCode` versus `getInviteCodes`.